docs/pages/core-concepts.mdx

```diff
@@ -19,6 +19,31 @@ A minimal Teleport cluster consists of the **Teleport Auth Service** and
 services from a single `teleport` process on a [Linux
 host](linux-demo.mdx).
 
+## Teleport services
+
+A **Teleport service** manages access to resources in your infrastructure, such
```

Comment:
I think by adding another term we might inadvertently be adding to the confusion. Now people are going to ask what the difference between a service and an agent service is.

I would say something like:

> Each Teleport process can run one or more services. Different services enable
> different Teleport functions. For example, the `ssh_service` provides SSH access
> to a resource, and the `database_service` proxies connections to databases.